salt/tops/varstack.py

```diff
@@ -45,27 +45,20 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
-# Import python libs
-import logging
-
 try:
     HAS_VARSTACK = False
     import varstack
     HAS_VARSTACK = True
 except ImportError:
     pass
 
-# Set up logging
-log = logging.getLogger(__name__)
-
 # Define the module's virtual name
 __virtualname__ = 'varstack'
```

Comment:
You don't need that.

Reply:
@isbm Why? The module does not use any logging, so I removed it.